### PR TITLE
Asynchronous Extension for the GetExceptional PHP Library

### PR DESCRIPTION
**Pull request by ktamas77**
I've created an asynchronous extension of the original PHP library.
I think in production environment it's mission critical to not rely on the network and log everything locally.
Cron (or other utility) can pick up these file based messages periodically & asynchronously and send to the remote.

This commit contains the extension itself, examples and the cron utility too.
I've made this extension in a non-intrusive way; the original library is 100% unmodified, however, the new library is 100% backward compatible with the original one.

Please feel free to use.

### CODE REVIEW

**Comment by benarent** [comment]
+1

**Comment by ktamas77** [comment]
should i close this request?

**Comment by benarent** [comment]
Hi @ktamas77 , Thanks for this. Since we've migrated Exceptional users to Airbrake http://blog.exceptional.io/bye-bye-exceptional-hello-airbrake I would recommend using https://github.com/airbrake/phpbrake/

**Comment by ktamas77** [comment]
Thank you, I'll close the ticket then.

File: exceptional/remote.php [hunk 1]
```diff
@@ -5,6 +5,7 @@ class ExceptionalRemote {
      * Does the actual sending of an exception
      */
     static function send_exception($exception) {
+        echo 'geci';
```
Comment (ktamas77) [inline exceptional/remote.php:8]: lol =)

**Pull request closed without merging**